www/include/common/javascript/tool.js

```diff
@@ -32,171 +32,166 @@
  * 
  */
 
-function checkItem(element, toCheck)
-{
-	if (element.type == 'checkbox') {
-		if (toCheck) {
-			element.checked = true;
-		} else {
-			element.checked = false;
-		}
-	} else if (element.type == 'radio') {
-		var element = document.Form[element.name];
-		var value = 0;
-		if (toCheck) {
-			value = 2;
-		}
-		for (var j = 0; j < element.length; j++) {
-			if (element[j].value == value) {
-				element[j].checked = true;
-			}
-		}
-	}
+function checkItem(element, toCheck) {
+  if (element.type == 'checkbox') {
+    if (toCheck) {
+      element.checked = true;
+    } else {
+      element.checked = false;
+    }
+  } else if (element.type == 'radio') {
+    var element = document.Form[element.name];
+    var value = 0;
+    if (toCheck) {
+      value = 2;
+    }
+    for (var j = 0; j < element.length; j++) {
+      if (element[j].value == value) {
+        element[j].checked = true;
+      }
+    }
+  }
 }
 
 function getChecked(element) {
-	if (element.type == 'checkbox') {
-		return element.checked;
-	}
-
-	var element = document.Form[element.name];
-	for (var j = 0; j < element.length; j++) {
-		if (element[j].checked) {
-			if (element[j].value > 0) {
-				return true;
-			}
-		}
-	}
-	return false;
+  if (element.type == 'checkbox') {
```

Comment:
I know it's just formatting but I think we can update this safely
```suggestion
  if (element.type === 'checkbox') {
```